test/jdk/javax/net/ssl/TLSCommon/TLSWithEdDSA.java

```diff
@@ -263,7 +263,7 @@ private static void keyCheck(PublicKey pubKey, String expPkType,
      *      initialization.
      */
     @Override
-    protected SSLContext createClientSSLContext() throws Exception {
+    public SSLContext createClientSSLContext() throws Exception {
```

Comment:
Why is this change needed, may be unintentional?

Reply:
reverted the file